### DISCUSSION TITLE
Spinning up child pods in a different namespace

### DISCUSSION THREAD

**Discussion by Vignesh-Kathirkamar** [nextflow-io/nextflow#5284] [Q&A] [answered]
Hi, I have a nextflow pipeline which would carry out six process and all these six processes have to run in a different namespace.

For now, I am spinning up the main pod (kind: Job) in a namespace called "dev" and all the six process within the .nf file is spinning up the pods in "dev" namespace only. But now I want the main pod to run in the "dev" namespace and other pods in a different namespace, say "ft1". 

I tried by changing the "k8s" spec within the config file as below:
```
k8s {
    namespace = 'ft1'
    serviceAccount = 'ft-ds'
}
```

but still got the error stating
``` 
User \"system:serviceaccount:dev-ds:dev\" cannot create resource \"pods\" in API group \"\" in the namespace \"ft1\"",
```

Is it possible to spin up the child pods alone in a different namespace?

**Comment by pditommaso** [comment]
Don't think the use of different namespaces is supported

**Reply by Vignesh-Kathirkamar** [reply]
As far as I have used nextflow, it's just spinning up the child pods in a specific order. So, why it's not possible to spin up those child pods in a different namespace.

**Reply by pditommaso** [reply]
because there's no way to specify the namespace at task level

**Reply by bentsherman** [reply, marked as answer]
I think he is only trying to run the head job in one namespace and all task pods in another. Should be possible as long as the head job namespace / service account has permission to submit pods to the other namespace

**Reply by Vignesh-Kathirkamar** [reply]
@bentsherman  thanks for the reply. This helps.
